test/data/convert/out/invoice-complete.xml

```diff
@@ -142,17 +142,22 @@
     </cac:TaxSubtotal>
   </cac:TaxTotal>
   <cac:LegalMonetaryTotal>
-    <cbc:LineExtensionAmount currencyID="EUR">1800.00</cbc:LineExtensionAmount>
-    <cbc:TaxExclusiveAmount currencyID="EUR">1801.00</cbc:TaxExclusiveAmount>
-    <cbc:TaxInclusiveAmount currencyID="EUR">2143.19</cbc:TaxInclusiveAmount>
+    <cbc:LineExtensionAmount currencyID="EUR">1710.00</cbc:LineExtensionAmount>
+    <cbc:TaxExclusiveAmount currencyID="EUR">1711.00</cbc:TaxExclusiveAmount>
+    <cbc:TaxInclusiveAmount currencyID="EUR">2036.09</cbc:TaxInclusiveAmount>
     <cbc:AllowanceTotalAmount currencyID="EUR">10.00</cbc:AllowanceTotalAmount>
     <cbc:ChargeTotalAmount currencyID="EUR">11.00</cbc:ChargeTotalAmount>
-    <cbc:PayableAmount currencyID="EUR">2143.19</cbc:PayableAmount>
+    <cbc:PayableAmount currencyID="EUR">2036.09</cbc:PayableAmount>
   </cac:LegalMonetaryTotal>
   <cac:InvoiceLine>
     <cbc:ID>1</cbc:ID>
     <cbc:InvoicedQuantity unitCode="HUR">20</cbc:InvoicedQuantity>
-    <cbc:LineExtensionAmount currencyID="EUR">1800.00</cbc:LineExtensionAmount>
+    <cbc:LineExtensionAmount currencyID="EUR">1710.00</cbc:LineExtensionAmount>
+    <cac:AllowanceCharge>
+      <cbc:ChargeIndicator>false</cbc:ChargeIndicator>
+      <cbc:MultiplierFactorNumeric>5</cbc:MultiplierFactorNumeric>
```

Comment:
Are you sure this is the right field? Multiplier Factor suggests to me something like "1.05" as opposed to 5...